testing/tests/DevExpress.ui.widgets/tabPanel.tests.js

```diff
@@ -627,6 +629,63 @@ QUnit.module('keyboard navigation', {
     }
 });
 
+
+QUnit.module('Disabled items', {
+    beforeEach() {
+        const items = [
+            { text: 'user', title: 'Personal Data' },
+            { text: 'comment', title: 'Contacts' },
+        ];
+
+        fx.off = true;
+
+        this.$element = $('#tabPanel').dxTabPanel({
+            focusStateEnabled: true,
+            items
+        });
+
+        this.instance = this.$element.dxTabPanel('instance');
+        this.$tabs = this.$element.find(toSelector(TABS_CLASS));
+
+        this.instance.option('items[1].disabled', true);
+        this.instance.focus();
+    },
+    afterEach() {
+        fx.off = false;
+    }
+}, () => {
+    QUnit.test('disabled item can be focused by keyboard', function(assert) {
+        const disabledItem = $(toSelector(TABS_ITEM_CLASS)).eq(1);
```

Comment:
It's a jquery element, so let's update the name to point on this.
It's also not necessary to wrap this element in `$()` on asserts.
```suggestion
        const $disabledItem = $(toSelector(TABS_ITEM_CLASS)).eq(1);
```

Reply:
Okay, a good point! I fixed it in commit [https://github.com/DevExpress/DevExtreme/pull/23944/commits/27401af212f62f32884086b9d8ad41b35504686c#diff-a9e0042294205504427bc9ec9f8fe30ff4bd2cca804419279738c1df277f183fR658]. Thanks!